deprecated/fabric-networking-v0/src/client/java/net/fabricmc/fabric/api/network/ClientSidePacketRegistry.java

```diff
@@ -23,7 +23,7 @@
 import net.minecraft.util.Identifier;
 import net.minecraft.network.PacketByteBuf;
 
-import net.fabricmc.fabric.impl.networking.ClientSidePacketRegistryImpl;
+import net.fabricmc.fabric.impl.client.networking.v0.ClientSidePacketRegistryImpl;
```

Comment:
I think this may be better as `net.fabricmc.fabric.impl.networking.v0.client`

Reply:
My name matches the current guidelines for package naming (and neither of these passed the old checkstyle)

Reply:
Yep, I was wrong. Thanks 👍